src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -99,6 +99,8 @@ internal static class IDEDiagnosticIds
         public const string ValueAssignedIsUnusedDiagnosticId = "IDE0059";
         public const string UnusedParameterDiagnosticId = "IDE0060";
 
+        public const string UseExpressionBodyForLocalFunctionsDiagnosticId = "IDE0061";
```

Comment:
probably worth a comment saying this conceptually belongs with the rest, but is here because it was added later.